core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+The Client DLT API as an external BPI API is implementation specific and will not be discussed further.
+
+**[R26]**	DLTs used in the implementation of a BPI MUST support bilateral and multi-lateral digital representations of legal contracts.
+
+**[R27]**	A DLT or BPI Abstraction Layer used in a BPI MUST support more than one DLT instance.
+
+**[R28]** A DLT or BPI Abstraction Layer used in a BPI MUST support more than one DLT type.
+
+This approach avoids lengthy discussions about which DLT protocol to utilize for a BPI, simplifying the decision making process considerably if most common DLTs are incorporated.
+
+Note that irrespective of whether one is in a public or private DLT scenario, the settings of a protocol such as block time, consensus model, type of execution framework etc. needs to be agreed upon by operating entities in some fashion either informally such as in Ethereum or formally such as in the Trade Finance consortium Komgo. 
+
+The agreement on the governance entity, its rules, and its method of achieving interval syn-chronization consensus, as well as the definition of acceptable governance structures and their rules is beyond the scope of this document.
+
+**[R29]**	The (commercial) counterparties MUST agree on the BPI.
+
+### 2.8.3	External Applications
+
+**[R30]**	Application/s providing commercial transaction functionality such as billing to (commercial) counterparties, and re, therefore, external with respect to the BPI, MUST be independent of any BPI.
+
+Note, that this requirement is motivated by reducing the dependency of counterparty internal systems on the BPI and vice versa. 
+
+### 2.8.4	Baseline Protocol Stack Detailed Reference Architecture Layers and Componants
+
+<figure>
+  <img
+  src="./images/High-Level-Baseline-Architecture.png"
+  >
+  <figcaption>Figure 3: Detailed Baseline Reference Architecture Layers and Components</figcaption>
+</figure>
+
+A Baseline Protocol Stack Reference Architecture as depicted above in Figure 3 is comprised of the following layers:
+* **Baseline Protocol (BPI) Abstraction Layer**: This layer enables accessing all externally available BPI functions through APIs as defined in the Baseline Protcol API Standards document 
+* **Middleware Layer**: This layer manages all counterparties to an agreement and its associated workflows and worksteps with business rules and business data as well as all counterparrty delegates. In addition, it manages all messaging between counterparties to an agreement and instantiation of processing layers based on newly created or updated agreements and their workflows, worksteps, business rules and business data.  
+* **Processing Layer**: Manages, properly sequences and deterministiaclly processes and finalizes in a privacy-preserving, cryptographically verifiable manner all state change requests from counterparties to all agreements represented in the BPI.  
```

Comment:
deterministically (sp)

add a comma after 'properly sequences' (I had to read it twice)